h2o-algos/src/main/java/hex/schemas/GLMModelV3.java

```diff
@@ -254,6 +253,22 @@ public int compare(Integer o1, Integer o2) {
     }
   } // GLMModelOutputV2
 
+  public static void calculateVarimpBase(double[] magnitudes, Integer[] indices, double[] betaNorm) {
+    for (int i = 0; i < magnitudes.length; ++i) {
+      magnitudes[i] = (float) betaNorm[i];
+      if (magnitudes[i] < 0) magnitudes[i] *= -1;
+    }
+    
+    Arrays.sort(indices, new Comparator<Integer>() {
```

Comment:
You might want to take a look at ArrayUtils#sort - it does the same thing: Sort an integer array of indices based on values

Most of this method won't be needed then.

Reply:
Good idea.  Will do!

Reply:
@michalk:  I check

Reply:
Done